### DISCUSSION TITLE
The  "press any key to boot from cd or dvd" don't show up

### DISCUSSION THREAD

**Discussion by WiTi34** [utmapp/UTM#5889] [UTM Help (macOS)]
I tried to install windows 11, on my macbook pro m1. I don't get the message that say "press any key to boot from cd or dvd"  when I do emulate with the turtle. But I can get it with virtualise with the rabbit. 
But i need to emulate to use some app like solidworks, so i'm stuck, i tried to get help on discord but no one respond. 

**Configuration**  
* UTM Version: the last one 
* macOS Version: 13.5.2
* Mac Chip (Intel, M1, ...): M1

**Comment by osy** [comment]
It seems like you chose the wrong architecture. When you were creating the VM, did it not warn you about using the wrong iso?

**Reply by WiTi34** [reply]
I, no, the same iso work virtualise, but the message don't show up on emulate 
<img width="707" alt="Capture d’écran 2023-11-12 à 17 37 04" src="https://github.com/utmapp/UTM/assets/49367554/1819e742-49ac-40e2-a814-691428a44e26">
nidi

**Reply by osy** [reply]
When you chose emulate you selected the wrong architecture. It should have told you this in app.

**Reply by WiTi34** [reply]
Ok
<img width="456" alt="Capture d’écran 2023-11-12 à 17 54 04" src="https://github.com/utmapp/UTM/assets/49367554/f906f404-a827-445e-8c55-7696435ea191">
, so in need to change this ?

**Comment by tifasoftware** [comment]
Yes set it to ARM64Sent from my iPhoneOn Nov 12, 2023, at 10:56 AM, 8i ***@***.***> wrote:﻿
Ok

, so in need to change this ?

**Reply by WiTi34** [reply]
well it don't do anything now 
<img width="826" alt="Capture d’écran 2023-11-12 à 23 56 42" src="https://github.com/utmapp/UTM/assets/49367554/4c024c84-c431-402f-85d5-091855145d2e">

**Comment by tifasoftware** [comment]
Start from a new virtual machine, this time select Virtualize >> WindowsOn Nov 12, 2023, at 4:57 PM, 8i ***@***.***> wrote:﻿
well it don't do anything now

**Reply by WiTi34** [reply]
yes it work like that, but to use some app like solidworks i need to emulate, or it can work on virtualize ?
